usr.sbin/devinfo/devinfo.c

```diff
@@ -83,21 +118,31 @@ print_resource(struct devinfo_res *res)
 	struct devinfo_rman	*rman;
 	bool			hexmode;
 	rman_res_t		end;
+	char			*safe_desc;
 
 	rman = devinfo_handle_to_rman(res->dr_rman);
 	hexmode =  (rman->dm_size > 1000) || (rman->dm_size == 0);
 	end = res->dr_start + res->dr_size - 1;
 
+	safe_desc = (char*) malloc(strlen(rman->dm_desc));
```

Comment:
cast to `char *` is useless here. `malloc()` errors should be checked here too

Reply:
Noted. See below.

Reply:
Fixed